lib/ai_bot/commands/search_command.rb

```diff
@@ -208,9 +208,14 @@ def process(**search_args)
           row = {
             title: post.topic.title,
             url: Discourse.base_path + post.url,
+            username: post.user&.username,
             excerpt: post.excerpt,
             created: post.created_at,
             category: category_names,
+            likes: post.like_count,
+            topic_views: post.topic.views,
```

Comment:
Just checking, is `post.topic` and `post.user` already preloaded with `includes` elsewhere to avoid N1?

Reply:
yeah we already had it for title.